### PR TITLE
Add iOS setup

### PR DESCRIPTION
**Pull request by Ronserruya**
Not a lot of content, just added the steps to use the SDK with iOS

### CODE REVIEW

**Comment by Ronserruya** [comment]
@chaseeb

**Review by chaseeb — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -44,11 +44,23 @@ android {
 ```
 
 
-## iOS Setup - Not supported yet
+## iOS Setup
 
-In the iOS Player Settings, the `Target minimum iOS Version` must be set to 8.1 or newer.
+1. You need to have [cocoapods](https://cocoapods.org/) installed
```
Comment (cnotethegr8) [inline README.md]: Spelled as 'CocoaPods'. Also add a period to the end of the sentence.

File: README.md [hunk 2]
```diff
@@ -44,11 +44,23 @@ android {
...
-Note: if you plan on doing any native iOS developement on the plugin, there are some changes that need to be made to the KinSDK, KinUtil and Sodium Xcode projects. They all need to have bitcode enabled and "build active achitectures only" set to no so that you can get debug symobols.
+2. In the iOS Player Settings, the `Target minimum iOS Version` must be set to 8.1 or newer.
+
+In addition, there is another step that needs to be done after you finish building your game.  
+Usually you would:  
+1. Build project
+2. Open the resulting .xporject with Xcode
+3. Build using Xcode
+
+To use the the Kin SDK do the following instead:
```
Comment (cnotethegr8) [inline README.md]: Remove the second 'the'.

File: README.md [hunk 3]
```diff
@@ -44,11 +44,23 @@ android {
 ```
 
 
-## iOS Setup - Not supported yet
+## iOS Setup
 
-In the iOS Player Settings, the `Target minimum iOS Version` must be set to 8.1 or newer.
+1. You need to have [cocoapods](https://cocoapods.org/) installed
```
Comment (cnotethegr8) [inline README.md]: Remove the space between items.

File: README.md [hunk 4]
```diff
@@ -44,11 +44,23 @@ android {
...
+
+In addition, there is another step that needs to be done after you finish building your game.  
+Usually you would:  
+1. Build project
+2. Open the resulting .xporject with Xcode
+3. Build using Xcode
+
+To use the the Kin SDK do the following instead:  
+1. Build project
+2. Run the file ```pods.command``` in the output folder
```
Comment (cnotethegr8) [inline README.md]: No need to do a triple \``` just do a single \` on each side. \`pods.command\`

**Review by cnotethegr8 — CHANGES_REQUESTED** [review]

**Review by cnotethegr8 — APPROVED** [review]

**Pull request merged**